<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QLabel" name="label_background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>480</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/background/background.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>422</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_timer">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>400</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>start</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_lline" native="true">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>80</y>
     <width>131</width>
     <height>321</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_lline_1">
    <property name="geometry">
     <rect>
      <x>68</x>
      <y>291</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/001.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_2">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>261</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/002.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_3">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>231</y>
      <width>32</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/003.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_4">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>200</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/004.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_5">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>169</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/005.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_6">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>140</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/006.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_7">
    <property name="geometry">
     <rect>
      <x>32</x>
      <y>110</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/007.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>82</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/008.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_9">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>52</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/009.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_lline_10">
    <property name="geometry">
     <rect>
      <x>63</x>
      <y>24</y>
      <width>32</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/lline/010.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_titlebar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="MQlabel" name="label_titlebar_2">
    <property name="geometry">
     <rect>
      <x>76</x>
      <y>4</y>
      <width>48</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/titlebar/001.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel" name="label_titlebar_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>8</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/titlebar/ready.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel" name="label_titlebar_3">
    <property name="geometry">
     <rect>
      <x>134</x>
      <y>0</y>
      <width>48</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/titlebar/002.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel" name="label_titlebar_4">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>48</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/titlebar/003.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel" name="label_titlebar_5">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>0</y>
      <width>40</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/titlebar/keyout.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel" name="label_titlebar_6">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/titlebar/pagoff.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel" name="label_titlebar_7">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>0</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/titlebar/power.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_number" native="true">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>19</y>
     <width>408</width>
     <height>408</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_numbre">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>408</width>
      <height>408</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/number/number1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_numbre_1">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>408</width>
      <height>368</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_numbre_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>408</width>
      <height>368</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="widget_center" native="true">
    <property name="geometry">
     <rect>
      <x>115</x>
      <y>156</y>
      <width>192</width>
      <height>72</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_rline" native="true">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>80</y>
     <width>155</width>
     <height>357</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_back">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>-4</y>
      <width>155</width>
      <height>357</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/rline/back.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_rline_1">
    <property name="geometry">
     <rect>
      <x>36</x>
      <y>20</y>
      <width>60</width>
      <height>130</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_rline_2">
    <property name="geometry">
     <rect>
      <x>28</x>
      <y>180</y>
      <width>71</width>
      <height>135</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_rline_1</zorder>
   <zorder>label_rline_2</zorder>
   <zorder>label_back</zorder>
  </widget>
  <widget class="QLabel" name="label_npdr">
   <property name="geometry">
    <rect>
     <x>376</x>
     <y>388</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/number/P.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="widget_statebar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>448</y>
     <width>800</width>
     <height>32</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_statebackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/statebar/statebar.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_1">
    <property name="geometry">
     <rect>
      <x>41</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/2.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_2">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/0.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_3">
    <property name="geometry">
     <rect>
      <x>61</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/1.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_4">
    <property name="geometry">
     <rect>
      <x>72</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_5">
    <property name="geometry">
     <rect>
      <x>97</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_6">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_7">
    <property name="geometry">
     <rect>
      <x>126</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_8">
    <property name="geometry">
     <rect>
      <x>135</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_9">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_10">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_11">
    <property name="geometry">
     <rect>
      <x>185</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_12">
    <property name="geometry">
     <rect>
      <x>195</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_13">
    <property name="geometry">
     <rect>
      <x>548</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_14">
    <property name="geometry">
     <rect>
      <x>559</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_15">
    <property name="geometry">
     <rect>
      <x>569</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_16">
    <property name="geometry">
     <rect>
      <x>584</x>
      <y>11</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_17">
    <property name="geometry">
     <rect>
      <x>691</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_18">
    <property name="geometry">
     <rect>
      <x>701</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_19">
    <property name="geometry">
     <rect>
      <x>711</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_20">
    <property name="geometry">
     <rect>
      <x>721</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="MQlabel_Number" name="label_state_21">
    <property name="geometry">
     <rect>
      <x>731</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/source/numberl/3.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="MQlabel" name="label_anquan_1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>40</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/number/anquan1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="MQlabel" name="label_anquan_2">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>380</y>
     <width>40</width>
     <height>48</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/source/number/anquan1.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>421</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:26pt; font-weight:600; color:#ff0000;&quot;&gt;QQ:609932088&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>label_background</zorder>
  <zorder>widget_lline</zorder>
  <zorder>widget_titlebar</zorder>
  <zorder>widget_number</zorder>
  <zorder>widget_rline</zorder>
  <zorder>pushButton_close</zorder>
  <zorder>pushButton_timer</zorder>
  <zorder>label_npdr</zorder>
  <zorder>widget_statebar</zorder>
  <zorder>label_anquan_1</zorder>
  <zorder>label_anquan_2</zorder>
  <zorder>label</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MQlabel</class>
   <extends>QLabel</extends>
   <header>mqlabel.h</header>
  </customwidget>
  <customwidget>
   <class>MQlabel_Number</class>
   <extends>QLabel</extends>
   <header>mqlabel_number.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
